db/migrate/20210207035043_add_instruments_count_to_dealers.rb

```diff
@@ -0,0 +1,5 @@
+class AddInstrumentsCountToDealers < ActiveRecord::Migration[5.2]
```

Comment:
@duguy8 would this be duplicating data?

Reply:
This is actually adding a column to the dealer that counts how many instruments it has! There is a line I added in the model, `cache_count, true` and that makes it so anytime an instrument object is made it auto increments that column in the dealer table

Reply:
We can ask Brian or Jamison because I feel like in order to get the number of instruments we could do an active record stated that says something like 'select count(*) from instruments where dealer_id = 7' and that'll give us the number of instruments

Reply:
Lets definitely bring that up in our project review! There is a way to do a `dealerinstance.instruments.size` that will show you how many instruments it has. I just had a hard time figuring out how make a statement for the whole dealer class that could sort them by how many instruments, extension 2